src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcLayout.scala.html

```diff
@@ -62,44 +65,35 @@
         pageLayout: Option[PageLayout => Html] = Some(fixedWidthPageLayout(_)),
         headerContainerClasses: String = Header.defaultObject.containerClasses,
         backLink: Option[BackLink] = None
-)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
+)(@deprecated("HmrcLayout is deprecated, use HmrcStandardLayout instead", "6.3.0") contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
 
 @require(!(backLinkUrl.isDefined && backLink.isDefined), "Use backLink or backLinkUrl, but not both")
 
-@headerBlock = {
-    @hmrcStandardHeader(
-        serviceUrl = serviceUrl,
-        signOutUrl = signOutUrl,
-        serviceName = serviceName,
-        userResearchBanner = userResearchBannerUrl.map(url => UserResearchBanner(url = url)),
-        phaseBanner = phaseBanner,
-        displayHmrcBanner = displayHmrcBanner,
-        additionalBannersBlock = additionalBannersBlock,
-        containerClasses = headerContainerClasses
-    )
-}
-
-@beforeContent = {
-    @if(beforeContentBlock.isDefined) {
-        @beforeContentBlock
-    } else {
-        @if(isWelshTranslationAvailable) {
-            @hmrcLanguageSelectHelper()
-        }
-        @{(backLink orElse backLinkUrl.map(url => BackLink(href = url, content = Text(messages("back.text")))))
-            .map(link => govukBackLink(link))}
-    }
-}
-
-@govukLayout(
+@hmrcStandardLayout(
```

Comment:
@ellamdav what do you think about HmrcStandardPage rather than HmrcStandardLayout - looks really good to me otherwise, feels like service / banners / layout / and page content at the top level is a good way to shuffle things

Reply:
renamed to `HmrcStandardPage`